### ISSUE TITLE
Cannot mix VS DX coils and VS WSHP coils in the same input file

### ISSUE THREAD

**Issue by EnergyArchmage** [NREL/EnergyPlus#4915]
A get input call has the wrong index variable resulting in a situation where models cannot have both Coil:Cooling:DX:VariableSpeed and Coil:Cooling:WaterToAirHeatPump:VariableSpeedEquationFit in the same input file.

line 771 in VariableSpeedCoils.cc

```
        GetObjectItem( CurrentModuleObject, DXCoilNum, AlphArray, NumAlphas, NumArray, NumNums, IOStat, lNumericBlanks, lAlphaBlanks, cAlphaFields, cNumericFields );
```

Use of DXCoilNum is wrong here, it includes the other types of coils such as VSWSHP that have already been setup in the same VarSpeedCoil structure.

The way the for loop is written it needs to be like this, (but WatertoAirHPNum is a poor name here) :
            GetObjectItem( CurrentModuleObject, WatertoAirHPNum, AlphArray, NumAlphas, NumArray, NumNums, IOStat, lNumericBlanks, lAlphaBlanks, cAlphaFields, cNumericFields );

**Comment by Myoldmopar** [comment]
@EnergyArchmage Is this included in the fixes in PR #5101 ?

**Comment by EnergyArchmage** [comment]
yes.

**Comment by mjwitte** [comment]
Closed via #5178
